### DISCUSSION TITLE
How to force an RDP session to use the glyph protocol to send text in window titles, including bold text？

### DISCUSSION THREAD

**Discussion by Mr-LZG** [FreeRDP/FreeRDP#10412] [Q&A]
I set the glyph capability to GLYPH_SUPPORT_ENCODE, but the problem is that the bold text in the window title is not sent using the glyph protocol. Do I need to disable or enable any capabilities during RDP capability negotiation?

**Comment by akallabeth** [comment]
@Mr-LZG please start reading (and understanding) the answers you´re getting:
1. There is no `glyph protocol` but an (ancient) mode for windows XP and older to do some font rendering client side. **THIS SHOULD NO LONGER BE USED** 
2. all systems starting with windows 7 no longer use the `GDI` graphics redirection (only in a very slow fallback mode) but
   * `rfx` graphics redirection (windows 7)
   * `gfx` pipeline (windows 8 and newer)

**Reply by Mr-LZG** [reply]
I want to capture the window title in the RDP remote desktop, so I want the RDP session to force the use of the glyph protocol. It seems that you cannot capture the text in the window title using rfx or gfx.

**Reply by akallabeth** [reply]
good luck with that.
you can not force it, some servers do not support it (depending on configuration) and fall back to bitmap transfer (huge bandwidth requirement, *very* slow screen updates, ...
